src/Microsoft.DotNet.Interactive.Documents.Tests/JupyterFormatTests.cs

```diff
@@ -983,6 +986,11 @@ public void serialized_notebook_has_appropriate_metadata()
                     name = "C#",
                     pygments_lexer = "csharp",
                     version = "10.0"
+                },
+                dotnet_interactive = new
+                {
+                    defaultKernelName = "csharp",
+                    items = Array.Empty<object>()
```

Comment:
if we default to `csharp`, then `items` should contain `{ name: "csharp", aliases: [] }`

Reply:
I notice we don't have `language` in this object. Did you hit that in your PR?